src/components/Jam.jsx

```diff
@@ -0,0 +1,320 @@
+  const INTERNAL_DEST_ACCOUNT = 0
+
+  useEffect(() => {
+    const abortCtrl = new AbortController()
+
+    setAlert(null)
+    setIsLoading(true)
+
+    const loadingServiceInfo = reloadServiceInfo({ signal: abortCtrl.signal }).catch((err) => {
+      const message = err.message || t('send.error_loading_wallet_failed')
+      !abortCtrl.signal.aborted && setAlert({ variant: 'danger', message })
+    })
+
+    const loadingWalletInfo = reloadCurrentWalletInfo({ signal: abortCtrl.signal }).catch((err) => {
+      const message = err.message || t('send.error_loading_wallet_failed')
+      !abortCtrl.signal.aborted && setAlert({ variant: 'danger', message })
+    })
+
+    Promise.all([loadingServiceInfo, loadingWalletInfo]).finally(() => !abortCtrl.signal.aborted && setIsLoading(false))
+
+    return () => abortCtrl.abort()
+  }, [reloadServiceInfo, reloadCurrentWalletInfo, t])
+
+  useEffect(() => {
+    const coinjoinInProgress = serviceInfo && serviceInfo.coinjoinInProgress
+    const makerRunning = serviceInfo && serviceInfo.makerRunning
+
+    setCollaborativeOperationRunning(coinjoinInProgress || makerRunning)
+  }, [serviceInfo])
+
+  const getNewAddresses = (count, mixdepth) => {
+    const externalBranch = walletInfo.data.display.walletinfo.accounts[mixdepth].branches.find((branch) => {
+      return branch.branch.split('\t')[0] === 'external addresses'
+    })
+
+    const newAddresses = []
+
+    externalBranch.entries.every((entry) => {
+      if (entry.status === 'new') {
+        newAddresses.push(entry.address)
+      }
+
+      if (newAddresses.length >= count) {
+        return false
+      }
+
+      return true
+    })
+
+    return newAddresses
+  }
+
+  const startSchedule = async (values) => {
+    if (collaborativeOperationRunning) {
+      return
+    }
+
+    const destinations = [values.dest1, values.dest2, values.dest3]
+
+    const body = { destination_addresses: destinations }
+
+    // Todo: Testing toggle is deactivated until https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1260 is merged.
+    // if (process.env.NODE_ENV === 'development' && useInsecureTestingSettings) {
+    //   body.tumbler_options = {
+    //     addrcount: 3,
+    //     minmakercount: 1,
+    //     makercountrange: [1, 1],
+    //     mixdepthcount: 3,
+    //     mintxcount: 2,
+    //     txcountparams: [1, 1],
+    //     timelambda: 0.1,
+    //     stage1_timelambda_increase: 1.0,
+    //     liquiditywait: 10,
+    //     waittime: 1.0,
+    //     mixdepthsrc: 0,
+    //   }
+    // }
+
+    try {
+      const res = await Api.postTumblerStart({ walletName: wallet.name, token: wallet.token }, body)
+
+      if (!res.ok) {
+        await Api.Helper.throwError(res, t('schedule.error_starting_schedule_failed'))
+      }
+
+      setCollaborativeOperationRunning(true)
+    } catch (err) {
+      setAlert({ variant: 'danger', message: err.message })
```

Comment:
Alert is never reset like it is done on other pages.

<img src="https://user-images.githubusercontent.com/3358649/167638355-82c4343b-5ca8-4c04-af86-fc4ebd845f99.png" width=400 />

Does not need to be fixed right away, can be an own GitHub Issue, just noted as reference.

In general, maybe solutions like toasts or overlay messages are better suited for this kind of information. What do you think?

Reply:
Fixed it here.  ✅